public/iiif/speculative-annotations-manifest.json

```diff
@@ -2610,7 +2610,7 @@
       ],
       "homepage": [
         {
-          "id": "https://www.loc.gov/item/nm0095/",
+          "id": "https://www.loc.gov/resource/hhh.nm0095.sheet/?sp=6",
```

Comment:
@jaimemears Just making sure the `id` is not changing to `hhh.nm0095` overall in the app?

Reply:
im not seeing a manifest at the item level (hhh.nm0095), just the collection level (nm0095), so based on that the id should probably stay nm0095.